src/form/Input.js

```diff
@@ -18,8 +18,10 @@
     Input.prototype._class += " form_Input";
     Input.prototype.implements(IInput.prototype);
 
-    Input.prototype.publish("type", "text", "set", "Input type", ["textbox", "number", "checkbox", "button", "select", "textarea", "date", "html-color"]);
+    Input.prototype.publish("type", "text", "set", "Input type", ["html-color", "number", "checkbox", "button", "select", "textarea", "date", "text", "range", "search", "email", "time", "datetime"]);
     Input.prototype.publish("selectOptions", [], "array", "Array of options used to fill a dropdown list");
+    Input.prototype.publish("min", null, "number", "Minimum value for Range input");
+    Input.prototype.publish("max", null, "number", "Maximum value for Range input");
```

Comment:
Should be low and high (to match the Slider PP) - also I would expect the data in/out to match the slider as well (it may mean the slider matches this version internally).